### DISCUSSION TITLE
Issue adding ACE1

### DISCUSSION THREAD

**Discussion by braxtonowens** [ACEsuit/ACE1.jl#35] [Help with Installation]
<img width="391" alt="image" src="https://user-images.githubusercontent.com/45109437/173928247-1bc0c351-feab-4fd2-8505-f0179dab13a0.png">

**Comment by cortner** [comment]
Can you create a cell at the top of your Jupiter notebook with 
```
using Pkg
Pkg.status()
```

**Reply by cortner** [reply]
(and post the output here)

**Reply by braxtonowens** [reply]
<img width="432" alt="image" src="https://user-images.githubusercontent.com/45109437/173929072-6c8f93dc-2aab-438b-9afa-4a1d280036bb.png">

**Reply by cortner** [reply]
ok and then in the cell underneath you run `using ACE1` and then output will be?

**Reply by braxtonowens** [reply]
Well, sorry. Inexplicably there are not any issues. Sorry for the inconvenience. Thanks for taking the time to assist.

**Reply by cortner** [reply]
no worries at all. I assume that you hadn't activated the right project before.

**Reply by cortner** [reply]
Please let me know if you run into any problems again.
